pandas/tests/series/test_datetime_values.py

```diff
@@ -409,3 +409,12 @@ def test_date_tz(self):
                            date(2015, 11, 22)])
         assert_series_equal(s.dt.date, expected)
         assert_series_equal(s.apply(lambda x: x.date()), expected)
+
+    def test_datetime_understood(self):
+        # Ensures it doesn't fail to create the right series reported in issue#16726
+        result = pd.Series(pd.date_range("2012-01-01", periods=3)) - pd.offsets.DateOffset(days=6)
+        expected = pd.Series(pd.to_datetime(['2011-12-26', '2011-12-27', '2011-12-28']))
+        tm.assert_series_equal(result, expected)
+
```

Comment:
These blank lines at the end caused the linting error https://travis-ci.org/pandas-dev/pandas/jobs/245637992#L1393

Looks like some lines are too long as well. You can test this locally with `flake8 pandas/tests/series/tests_datetime_values.py` (might have to install flake8 first)`

Seems like the test passed though, so the style stuff should be the only change, then we can get this merged.